scripts/travis/publish-tag.sh

```diff
@@ -65,8 +65,18 @@ fi
 while sleep 9m; do echo "[Ping] Keeping Travis job alive ($((SECONDS / 60)) minutes)"; done &
 WAITER_PID=$!
 
-# Build and publish to Bintray
-echo "All checks passed, attempting to publish Rest.li $VERSION to Bintray..."
+# TODO: This just publishes to the test-repo for now, delete the Bintray publish step once no longer testing
+# Publish to JFrog Artifactory
+echo "All checks passed, attempting to publish Rest.li $VERSION to JFrog Artifactory..."
+./gradlew artifactoryPublish
```

Comment:
So looks like there is a task publishing to bintray and this new task publish to jfrog, are we going to have 2 for temp and eventually publish to jfrog (sounds like a migration process)

Reply:
Yeah, after this change, the process will be (1) publish to Bintray as usual and (2) publish to a test repo in JFrog.
I will do a follow-up change, after which the process will be (1) don't publish to Bintray at all and (2) publish to JFrog.

The reason for doing this in 2 steps is to confirm that the process works before cutting off Bintray (since that would block our release process).